spec/web/controllers/tasks/index_spec.rb

```diff
@@ -155,6 +155,41 @@
           expect(action.tasks.map(&:status)).to all(eq('in progress'))
         end
       end
+
+      context 'when are on the first page' do
```

Comment:
~~let's simplify specs here. For this we can use DI ([dependency injection](https://medium.com/@Bakku1505/introduction-to-dependency-injection-in-ruby-dc238655a278) and [wiki](https://en.wikipedia.org/wiki/Dependency_injection)) for repository instance:~~

So, I found a problem in pagination https://github.com/davydovanton/hanami-pagination/issues/10, thanks for this! 💯 🌟 🎉